gosyntect.go

```diff
@@ -90,6 +118,66 @@ type Client struct {
 	syntectServer string
 }
 
+// sharedClient is a shared client aross all HighlightCSSTable requests
+// so connections to the server can be reused.
+var sharedClient = &http.Client{Transport: &nethttp.Transport{}}
+
+// HighlightCSSTable performs a query to highlight code and return it in a line-numbered,
+// formatted table with CSS classes for spans
+func (c *Client) HighlightCSSTable(ctx context.Context, q *CSSTableQuery) (*Response, error) {
```

Comment:
Quite a bit of duplicate code here with few options to reduce it. I opted to have an entirely separate method to preserve backwards compatibility, but at the price of duplication.